templates/tree-editor/tree/tree-editor-widget.ts

```diff
@@ -12,43 +12,45 @@ import { ResourceTreeEditorWidget } from  'theia-tree-editor';
 
 @injectable()
 export class TreeEditorWidget extends ResourceTreeEditorWidget {
-  constructor(
-    @inject(MasterTreeWidget)
-    readonly treeWidget: MasterTreeWidget,
-    @inject(DetailFormWidget)
-    readonly formWidget: DetailFormWidget,
-    @inject(WorkspaceService)
-    readonly workspaceService: WorkspaceService,
-    @inject(ILogger) readonly logger: ILogger,
-    @inject(NavigatableTreeEditorOptions)
-    protected readonly options: NavigatableTreeEditorOptions,
-    @inject(DefaultResourceProvider)
-    protected provider: DefaultResourceProvider,
-    @inject(TreeEditor.NodeFactory)
-    protected readonly nodeFactory: TreeEditor.NodeFactory
-  ) {
-    super(
-      treeWidget,
-      formWidget,
-      workspaceService,
-      logger,
-      TreeEditorWidget.WIDGET_ID,
-      options,
-      provider,
-      nodeFactory,
-    );
-  }
-
-  protected getTypeProperty() {
-    return "typeId";
-  }
-
-  protected configureTitle(title: Title<Widget>): void {
-    super.configureTitle(title);
-    title.iconClass = 'fa fa-tree dark-purple';
-  }
+    constructor(
+        @inject(MasterTreeWidget)
+        readonly treeWidget: MasterTreeWidget,
+        @inject(DetailFormWidget)
+        readonly formWidget: DetailFormWidget,
+        @inject(WorkspaceService)
+        readonly workspaceService: WorkspaceService,
+        @inject(ILogger) readonly logger: ILogger,
+        @inject(NavigatableTreeEditorOptions)
+        protected readonly options: NavigatableTreeEditorOptions,
+        @inject(DefaultResourceProvider)
+        protected provider: DefaultResourceProvider,
+        @inject(TreeEditor.NodeFactory)
+        protected readonly nodeFactory: TreeEditor.NodeFactory
+    ) {
+        super(
+            treeWidget,
+            formWidget,
+            workspaceService,
+            logger,
+            TreeEditorWidget.WIDGET_ID,
+            options,
+            provider,
+            nodeFactory,
+        );
+    }
+          
+    protected getTypeProperty() {
+        return "typeId";
+    }
+    
+    protected configureTitle(title: Title<Widget>): void {
+        super.configureTitle(title);
+        title.iconClass = 'fa fa-coffee dark-purple';
+    }
 }
+
 export namespace TreeEditorWidget {
-  export const WIDGET_ID = 'json-forms-tree-editor';
-  export const EDITOR_ID = 'com.eclipsesource.tree.editor';
+    export const WIDGET_ID = 'my-tree-editor';
```

Comment:
these ids could be easily scoped to the name the user defines in yeoman, right?

Reply:
yes :)